Sources/Rugby/Commands/Cache/Other/Checksums/ChecksumsProvider.swift

```diff
@@ -30,17 +30,30 @@ struct Checksum {
 
 final class ChecksumsProvider {
     private let podsProvider = PodsProvider.shared
-    private var cachedChecksums: [String: Checksum]?
+    private static var cachedChecksums: [String: Checksum] = [:]
 
-    func getChecksums(forPods pods: Set<String>) throws -> [Checksum] {
-        let selectedCachedChecksums = pods.compactMap { cachedChecksums?[$0] }
-        if selectedCachedChecksums.count == pods.count { return selectedCachedChecksums }
+    private var shouldChecksumContent: Bool
```

Comment:
We can use let here:
```suggestion
    private let shouldChecksumContent: Bool
```